packages/scss/src/commons/utils/reset.scss

```diff
@@ -1,7 +1,7 @@
 @mixin list($suffix: '', $list: 'ul') {
 	margin: 0 #{$suffix};
 
-	@if $list == ('ul' or 'ol') {
+	@if $list == 'ul' or 'ol' {
```

Comment:
Is a string always truthy or falsy?

```scss
@mixin list($suffix: '', $list: 'ul') {
  @debug('===========================');
  @debug($list);
  @debug($list == ('ul' or 'ol'));
  @debug($list == 'ul' or 'ol');
  @debug($list == 'ul' or $list == 'ol');
  @debug('ol' == true);
  @debug('ol' == false);
  @debug('===========================');
}

@include list;
@include list($list: 'ol');
@include list($list: 'anything');
```
gives me
```
DEBUG: ===========================
DEBUG: ul
DEBUG: true
DEBUG: true
DEBUG: true
DEBUG: false
DEBUG: false
DEBUG: ===========================
DEBUG: ===========================
DEBUG: ol
DEBUG: false
DEBUG: ol
DEBUG: true
DEBUG: false
DEBUG: false
DEBUG: ===========================
DEBUG: ===========================
DEBUG: anything
DEBUG: false
DEBUG: ol
DEBUG: false
DEBUG: false
DEBUG: false
DEBUG: ===========================
```

Reply:
```scss
  @if 'ol' {
    @debug 'A string is truthy.';
  } @else {
    @debug 'A string is falsy.';
  }
```
gives me
```
A string is truthy.
```

Which seems to indicate the expression would always be truthy.

Reply:
The solution proposed by @ymalie-lucca  works: https://gist.github.com/vvalentin-lucca/3ac672d32d12a535eb4b01a7ad54fe4a

Reply:
Should `dl` have both the `padding` and the `list-style-type properties?

Reply:
Ah gosh no!

Reply:
`@if $list == 'dl' {`?

Reply:
`$list == 'ul' or $list == 'ol'` works.

Reply:
https://gist.github.com/vvalentin-lucca/ad36b64b2ca84979bd956e14d93f79d1